src/pybrake/middleware/aiohttp.py

```diff
@@ -2,7 +2,7 @@
 
 from aiohttp import web
 
-from .notifier import Notifier
+from .. import Notifier
```

Comment:
Just curious, what's the meaning of this?

Reply:
It denotes the import class's relative route. When I add more middleware, I'll consider changing it. Because in other middleware, we don't use the Notifier class to construct instances within middleware. To initialise middleware in other middleware, we must explicitly pass an instance of Notifier.